### PR TITLE
Update French Translation

### PR DESCRIPTION
**Pull request by grenagit**

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -1,40 +1,78 @@
...
-    <string name="digiview_feed">DigiView\nfeed</string>
-    <string name="waiting_for_usb_device">En attente du périphérique USB…</string>
+    <string name="digiview_feed">DigiView\nflux</string>
+    <string name="waiting_for_usb_device">Veuillez connecter les lunettes</string>
     <string name="usb_device_approved">Périphérique USB approuvé</string>
-    <string name="usb_device_detached_waiting">Périphérique USB déconnecté\nEn attente du périphérique USB…</string>
-    <string name="usb_device_found">Périphérique USB trouvé.</string>
+    <string name="usb_device_detached_waiting">Lunettes déconnectées</string>
+    <string name="usb_device_found">Lunettes connectées</string>
+    <string name="waiting_f    <string name="digiview_feed">DigiView\nflux</string>
```
Comment (fichek) [inline app/src/main/res/values-fr/strings.xml]: Error here.

**Review by fichek — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -1,40 +1,41 @@
...
+    <string name="about">À Propos</string>
+    <string name="app_Version">Version de l\'application</string>
     <string name="copyright">Copyright</string>
-    <string name="open_source_license">License Open-Source</string>
-    <string name="open_source_license_summary">License MIT</string>
-</resources>
\ No newline at end of file
+    <string name="open_source_license">Licence Open-Source</string>
+    <string name="open_source_license_summary">Licence MIT</string>
+</resources>>
```
Comment (fichek) [inline app/src/main/res/values-fr/strings.xml]: Double closing `>`

**Review by fichek — DISMISSED** [review]

**Comment by trigger7** [comment]
I had a quick look through grenagit's translation, looks good to me.
Much more faithful to the English strings than the previous ones.

**Comment by jlucidar** [comment]
Sorry for the delay. I just did a minor fix to replace "Analytiques" which sounded weird. (I'm french btw )
I sort of hesitated to merge this because the translations were already ok, but they are now closest to the english reference.

Thanks for the PR !

**Pull request merged**